winit/src/application.rs

```diff
@@ -459,17 +474,7 @@ fn build_user_interface<'a, A: Application>(
     user_interface
 }
 
-fn to_physical(size: winit::dpi::LogicalSize, dpi: f64) -> (u16, u16) {
-    let physical_size = size.to_physical(dpi);
-
-    (
-        physical_size.width.round() as u16,
-        physical_size.height.round() as u16,
-    )
-}
-
 // As defined in: http://www.unicode.org/faq/private_use.html
-// TODO: Remove once https://github.com/rust-windowing/winit/pull/1254 lands
 fn is_private_use_character(c: char) -> bool {
```

Comment:
Should this be removed?

Reply:
I prefer to keep this for now. `winit` is not filtering exactly the same characters.